torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -69,15 +69,15 @@ def create_runtime_wrapper(
     keep_input_mutations: bool,
     disable_amp: bool,
 ):
-    num_tokens = len(runtime_metadata.tokens)
-
     if not hasattr(compiled_fn, "_boxed_call"):
         compiled_fn = make_boxed_func(compiled_fn)
 
     def runtime_wrapper(*args):
+        num_tokens = len(runtime_metadata.tokens)
```

Comment:
hmmm - at runtime, if `config.unlift_effect_tokens`, doesn't that mean that we shouldn't need to pass in any extra arguments (the `None`s below)?

I guess one downside doing this is that the runtime wrapper gets more complicated (the calling convention changes depending on config - so we should probably have separate tests exercising the runtime wrapper for when the config is not set).

Although in the common case (when we're using inductor), fewer calling convention changes at runtime seems better.

Reply:
>  if config.unlift_effect_tokens is set to True, doesn't that mean that we shouldn't need to pass in any extra arguments (the Nones below)?

Yup that's true, but num_tokens will be 0 either way so it's the same behavior, but I can add an additional check.

Reply:
> so we should probably have separate tests exercising the runtime wrapper for when the config is not set)

Yeah I think right now the `torch.compile(backend="aot_eager")` cases and normal `export`/`aot_export_module` calls will exercise this path.

Reply:
> Yeah I think right now the torch.compile(backend="aot_eager") cases and normal export/aot_export_module calls will exercise this path.

Got it, this makes sense. Do you also mind updating the `aot_eager_decomp_partition` backend to set this config to be the same as inductor? That backend is morally meant to be "all the inductor decomps / configs, but without actually doing inductor codegen": https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/backends/debugging.py#L83

Reply:
> Yup that's true, but num_tokens will be 0 either way so it's the same behavior, but I can add an additional check.

Ah this makes sense. Then yeah let's just add an assert that `num_tokens` is 0 when this config is set